integration_tests/test_stage_package_version.py

```diff
@@ -27,6 +27,6 @@ def test_stage_package_gets_version(self):
             self.run_snapcraft, 'pull', 'stage-package-version')
         self.assertIn(
             "Error downloading stage packages for part 'hello': "
-            "The Ubuntu package 'hello=x.y-z' was not found.",
+            "The package 'hello=x.y-z' was not found.",
```

Comment:
Nope, this is not enforced, implemented purely in the `Ubuntu` class and the `.` syntax can be implemented in whatever the rpm one is called.